### YamlMime:ManagedReference
items:
- uid: System.IO.Log.SequenceFullException
  id: SequenceFullException
  children:
  - System.IO.Log.SequenceFullException.#ctor
  - System.IO.Log.SequenceFullException.#ctor(System.String)
  - System.IO.Log.SequenceFullException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.IO.Log.SequenceFullException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: SequenceFullException
  nameWithType: SequenceFullException
  fullName: System.IO.Log.SequenceFullException
  type: Class
  summary: "Teljes naplórekordokat sorozata esetén fordul elő, amelyek jelöli."
  syntax:
    content: 'public class SequenceFullException : Exception'
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.IO.Log.SequenceFullException.#ctor
  id: '#ctor'
  parent: System.IO.Log.SequenceFullException
  langs:
  - csharp
  name: SequenceFullException()
  nameWithType: SequenceFullException.SequenceFullException()
  fullName: System.IO.Log.SequenceFullException.SequenceFullException()
  type: Constructor
  assemblies:
  - System.IO.Log
  namespace: System.IO.Log
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IO.Log.SequenceFullException&quot;> </xref> osztály."
  syntax:
    content: public SequenceFullException ();
    parameters: []
  overload: System.IO.Log.SequenceFullException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.Log.SequenceFullException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.IO.Log.SequenceFullException
  langs:
  - csharp
  name: SequenceFullException(String)
  nameWithType: SequenceFullException.SequenceFullException(String)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(String)
  type: Constructor
  assemblies:
  - System.IO.Log
  namespace: System.IO.Log
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IO.Log.SequenceFullException&quot;> </xref> osztály egy adott hiba miatt."
  syntax:
    content: public SequenceFullException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
  overload: System.IO.Log.SequenceFullException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.Log.SequenceFullException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.IO.Log.SequenceFullException
  langs:
  - csharp
  name: SequenceFullException(SerializationInfo,StreamingContext)
  nameWithType: SequenceFullException.SequenceFullException(SerializationInfo,StreamingContext)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.IO.Log
  namespace: System.IO.Log
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IO.Log.SequenceFullException&quot;> </xref> a szerializált adatok osztályt."
  syntax:
    content: protected SequenceFullException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: info
      type: System.Runtime.Serialization.SerializationInfo
      description: "A <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> , amely tartalmazza a kivétel lépett fel az éppen szerializált objektum adatait."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "A <xref:System.Runtime.Serialization.StreamingContext>, amely a cél- és környezeti információkat tartalmaz.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.IO.Log.SequenceFullException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.IO.Log.SequenceFullException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.IO.Log.SequenceFullException
  langs:
  - csharp
  name: SequenceFullException(String,Exception)
  nameWithType: SequenceFullException.SequenceFullException(String,Exception)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(String,Exception)
  type: Constructor
  assemblies:
  - System.IO.Log
  namespace: System.IO.Log
  summary: "Egy új példányát inicializálja a <xref href=&quot;System.IO.Log.SequenceFullException&quot;> </xref> megadott hibaüzenet, és hivatkozni kell a belső kivételben találhatók okozza-e ezt a kivételt, a osztályt."
  syntax:
    content: public SequenceFullException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "Az üzenet, amely leírja a hibát."
    - id: inner
      type: System.Exception
      description: "A kivétellel, hogy a jelenlegi kivétel okozza."
  overload: System.IO.Log.SequenceFullException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.IO.Log.SequenceFullException.#ctor
  parent: System.IO.Log.SequenceFullException
  isExternal: false
  name: SequenceFullException()
  nameWithType: SequenceFullException.SequenceFullException()
  fullName: System.IO.Log.SequenceFullException.SequenceFullException()
- uid: System.IO.Log.SequenceFullException.#ctor(System.String)
  parent: System.IO.Log.SequenceFullException
  isExternal: false
  name: SequenceFullException(String)
  nameWithType: SequenceFullException.SequenceFullException(String)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.IO.Log.SequenceFullException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.IO.Log.SequenceFullException
  isExternal: false
  name: SequenceFullException(SerializationInfo,StreamingContext)
  nameWithType: SequenceFullException.SequenceFullException(SerializationInfo,StreamingContext)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.IO.Log.SequenceFullException.#ctor(System.String,System.Exception)
  parent: System.IO.Log.SequenceFullException
  isExternal: false
  name: SequenceFullException(String,Exception)
  nameWithType: SequenceFullException.SequenceFullException(String,Exception)
  fullName: System.IO.Log.SequenceFullException.SequenceFullException(String,Exception)
- uid: System.IO.Log.SequenceFullException.#ctor*
  parent: System.IO.Log.SequenceFullException
  isExternal: false
  name: SequenceFullException
  nameWithType: SequenceFullException.SequenceFullException
